changes.go

```diff
@@ -227,9 +227,7 @@ func (c *pollChangeManager) getChanges(changesUri *url.URL) error {
 	 */
 	if lastSequence != "" &&
 		getChangeStatus(lastSequence, resp.LastSequence) != 1 {
-		return changeServerError{
-			Code: "Ignore change, already have newer changes",
-		}
+		return nil
```

Comment:
The debug statement is already printed at the the parent function. That is why its not here.